packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -118,7 +118,8 @@ module.exports = {
           // Point ESLint to our predefined config.
           options: {
             configFile: path.join(__dirname, '../.eslintrc'),
-            useEslintrc: false
+            useEslintrc: false,
+            cache: true
```

Comment:
Babel [falls back to OS tmp dir](https://github.com/babel/babel-loader/blob/931c6195935dc068a65830746c86569500df6e68/src/fs-cache.js#L174). Does eslint-loader, too? We should probably use consistent mechanisms. If eslint-loader doesn't, we should send a PR.